### PR TITLE
Support dynamic non-root user for alluxio fuse

### PR DESCRIPTION
**Pull request by cheyang**

### CODE REVIEW

File: integration/docker/Dockerfile.fuse [hunk 1]
```diff
@@ -18,6 +18,7 @@
 FROM alpine:3.10.2 AS extractor
 # Note that downloads for *-SNAPSHOT tarballs are not available
 ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.4.0-SNAPSHOT/alluxio-2.4.0-SNAPSHOT-bin.tar.gz
+ARG ENABLE_DYANMIC_USER=false
```
Comment (apc999) [inline integration/docker/Dockerfile.fuse:21]: ENABLE_DYNAMIC_USER

File: integration/docker/Dockerfile.fuse [hunk 2]
```diff
@@ -29,11 +30,16 @@ RUN cd /opt && \
 # Remove the UFS libraries from the container, as the fuse daemon doesn't need them
 RUN rm -rf /opt/alluxio/lib/*
 
+RUN if [ ${ENABLE_DYANMIC_USER} = "true" ] ; then \
```
Comment (apc999) [inline integration/docker/Dockerfile.fuse:33]: ENABLE_DYNAMIC_USER

File: integration/docker/Dockerfile.fuse [hunk 3]
```diff
@@ -69,6 +75,15 @@ COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} --from=extractor /opt/ /opt/
 COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} conf /opt/alluxio/conf/
 COPY --chown=${ALLUXIO_USERNAME}:${ALLUXIO_GROUP} entrypoint.sh /
 
+RUN if [ ${ENABLE_DYANMIC_USER} = "true" ] ; then \
```
Comment (apc999) [inline integration/docker/Dockerfile.fuse:78]: ENABLE_DYNAMIC_USER

**Review by apc999 — CHANGES_REQUESTED** [review]

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
@jiacheliu3  can you take a look? thanks

File: integration/docker/Dockerfile.fuse [hunk 4]
```diff
@@ -18,6 +18,7 @@
 FROM alpine:3.10.2 AS extractor
 # Note that downloads for *-SNAPSHOT tarballs are not available
 ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.4.0-SNAPSHOT/alluxio-2.4.0-SNAPSHOT-bin.tar.gz
+ARG ENABLE_DYNAMIC_USER=false
```
Comment (jiacheliu3) [inline integration/docker/Dockerfile.fuse:21]: Please add the same comment and printout as the Dockerfile.

**Review by jiacheliu3 — APPROVED** [review]
LGTM with a request of change on the comment/printout. Thanks for the contribution!

File: integration/docker/Dockerfile.fuse (hunk 4 above)
Comment (apc999) [reply integration/docker/Dockerfile.fuse:22]: fixed

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3-fuse please

**Comment by apc999** [comment]
@cheyang  I have manually cherry-picked this patch to branch-2.3 and branch-2.3-fuse, for your information

**Pull request merged**